sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyAlert/GetAnomaliesForAlertOptions.cs

```diff
@@ -1,11 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Threading;
+
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
-    /// The set of options that can be specified when calling <see cref="MetricsAdvisorClient.GetAnomaliesForAlert"/>
-    /// or <see cref="MetricsAdvisorClient.GetAnomaliesForAlertAsync"/> to configure the behavior of the request.
+    /// The set of options that can be specified when calling <see cref="MetricsAdvisorClient.GetAnomalies(string, string, GetAnomaliesForAlertOptions, CancellationToken)"/>
+    /// or <see cref="MetricsAdvisorClient.GetAnomaliesAsync(string, string, GetAnomaliesForAlertOptions, CancellationToken)"/> to configure the behavior of the request.
```

Comment:
Is it valuable to include the whole signature instead of just saying the name of the method?
I am just wondering if in the future there is another overload that still needs `GetAnomaliesFroAlertOptions, this docstring is going to be big, and possibly confusing

Reply:
I guess same comment in other classes/properties

Reply:
If you take a look at the generated docs for the first beta ([github.io](https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Azure.AI.MetricsAdvisor/1.0.0-beta.1/api/Azure.AI.MetricsAdvisor.Models/Azure.AI.MetricsAdvisor.Models.GetAnomaliesForAlertOptions.html) and [docs.microsoft.com](https://docs.microsoft.com/en-us/dotnet/api/azure.ai.metricsadvisor.models.getanomaliesforalertoptions?view=azure-dotnet-preview)), you'll see the whole signature is added anyway. IntelliSense behaves the same way, so hiding it won't make it clearer for users.

Regarding your original question, we need to specify the whole signature since it's a code reference. It could end up linking to the wrong overload otherwise. We get a compilation-time error (suppressible) if we only add the method name.